src/logic/schedule-parser/schedule.parser.ts

```diff
@@ -35,6 +35,7 @@ export class ScheduleParser implements ScheduleProvider {
     let children;
     let nurses;
     let babysiter;
+    let dayWorkers;
 
     const metadataRaw = rawSchedule.find((r) => r[0][0].toLowerCase().trim() === "dni miesiąca");
```

Comment:
you change the rawSchedule, we have to change also this,
metadataRaw was before [["dni miesiąca" , 1 ,2 3,4 ...]]
now:
[["dni miesiąca ...],
["pracownicy dzienni ...],
["dzieci" ...]]

this should be adressed